src/UI/button/Button.scss

```diff
@@ -0,0 +1,10 @@
+.Btn {
+  padding: 5px 15px;
+  color: rgb(0, 173, 173);
```

Comment:
constant? you can create a palette and use it everywhere. If you have to switch themes it will be very difficult for you as it will require a ton of refactoring